src/dfcx_scrapi/core/test_cases.py

```diff
@@ -433,3 +433,153 @@ def calculate_coverage(self, coverage_type: int, agent_id: str = None):
         )
         response = client.calculate_coverage(request)
         return response
+
+    def get_covered_intents(
```

Comment:
@hkhaitan1 The following methods are actually duplicative code:

- `get_covered_intents`
- `get_covered_pages`
- `get_covered_route_groups`

The functionality for each of these is already present in the method [calculate_coverage](https://github.com/GoogleCloudPlatform/dfcx-scrapi/blob/main/src/dfcx_scrapi/core/test_cases.py#L402) using the parameter `coverage_type`.  

As mentioned in the docstring:
```
coverage_type: The type of coverage requested.
            INTENT = 1
            PAGE_TRANSITION = 2
            TRANSITION_ROUTE_GROUP = 3
```

Reply:
@hkhaitan1 I reviewed this some more and I think that all of the code you have should actually be implemented on the client side of things. There is no new functionality that is being created for Test Cases here, only the repurposing of code to transform into different output types.

If you had devised a novel way of presenting this data, say an "all-in-one report" or something like that, then we could consider adding it. But as it stands, this appears to just create some short hand methods of `calculate_coverage`.

Let's go ahead and discard these changes and focus on the Agent Assist changes for this particular PR.

Reply:
I had written a method to aggregate and write everything into a BQ table. But I will remove it for now.
I can re-purpose the existing code and open a new PR later.